### DISCUSSION TITLE
Libraries to implement multiplayer

### DISCUSSION THREAD

**Discussion by Verzach3** [HaxeFlixel/flixel#3223] [Q&A]
I was looking for libraries to implement multiplayer in a small game I want to make and I did not find any that was currently maintained, it is good that it is only on the PC platform, it is good that they are network libraries in general and then try to make an implementation with HxBit or something, so I'm open to alternatives and advice.

**Comment by Geokureli** [comment]
I used [Colyseus](https://docs.colyseus.io/getting-started/haxe-client/) in the past and found it pretty easy to make simple multiplayer games. I'm not great networking so I was pretty surprised that i got it working
